### PR TITLE
Feature/detailed route view

### PR DESCRIPTION
**Pull request by PaigeVannelli**
## Is this a fix or a feature?
Feature - Build out the Detailed Route Component 

## What is the change?
This component renders when the user clicks on a specific route. It shows the user driver details, days of the week and time the drive happens, and a distance from both the origin and destination. It should also allow the user to request a hitch with that specific user. It also then has all of the styling associated with the component including a profile photo and screenshots of maps 

## What does it fix?
n/a

## Where should the reviewer start?
Review the DetailedRoute component and the styling. 

## How should this be tested?
open the app and navigate to detailed-route page

## Relevant ScreenShots
n/a

### CODE REVIEW

File: src/components/App/App.tsx [hunk 1]
```diff
@@ -60,6 +61,14 @@ const App = () => {
...
           render={() => <Registration {...formProps} />}
         />
+        <Route
+            exact path='/:id'
+            render={({ match }) => {
+              return (
+                <DetailedRoute />
+              )
+            }}
+          />
```
Comment (alexthompson207) [inline src/components/App/App.tsx:71]: Good use of match here!

File: src/components/DetailedRoute/DetailedRoute.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+//   id: string
+// }
+
+// const DetailedRoute: FC<DetailedRouteProps> = () => {
+const DetailedRoute = () => {
+
+  const days = detailedRouteData.route.days.map(day => {
+    return (
+      <div className='day'>
+        {(day === 'tuesday' || day === 'thursday') ? day.charAt(0).toUpperCase() + day.slice(1,2) : day.substring(0, 1).toUpperCase()}
```
Comment (alexthompson207) [inline src/components/DetailedRoute/DetailedRoute.tsx:18]: Maybe we can add a conditional for Saturday and Sunday here

File: src/components/DetailedRoute/DetailedRoute.tsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+import userPhoto from '../../assets/images/man.png';
+import exampleMapOne from '../../assets/images/exampleMapOne.png'
+import exampleMapTwo from '../../assets/images/exampleMapTwo.png'
+
+// interface DetailedRouteProps {
+//   id: string
+// }
+
+// const DetailedRoute: FC<DetailedRouteProps> = () => {
+const DetailedRoute = () => {
```
Comment (alexthompson207) [inline src/components/DetailedRoute/DetailedRoute.tsx:13]: Shoud we add a type here

**Review by alexthompson207 — APPROVED** [review]
Nice work Page, the styling on this view looks soo good!

File: src/components/DetailedRoute/DetailedRoute.tsx (hunk 2 above)
Comment (itsnameissteven) [reply src/components/DetailedRoute/DetailedRoute.tsx:18]: agreed

**Review by itsnameissteven — APPROVED** [review]
Great job. I can't wait to get this hooked up!

**Pull request merged**